main.go

```diff
@@ -104,6 +105,20 @@ func main() {
 	flag.BoolVar(&enableLeaderElection, "leader-elect", false,
 		"Enable leader election for controller manager. "+
 			"Enabling this will ensure there is only one active controller manager.")
+	utils.StringFlagOrEnv(&constants.TrillianLogSignerImage, "trillian-log-signer-image", "TRILLIAN-LOG-SIGNER-IMAGE", constants.TrillianLogSignerImage, "The image used for trillian log signer.")
```

Comment:
please use `_` character instead of `-` when separating words in ENV variable name
```suggestion
	utils.StringFlagOrEnv(&constants.TrillianLogSignerImage, "trillian-log-signer-image", "TRILLIAN_LOG_SIGNER_IMAGE", constants.TrillianLogSignerImage, "The image used for trillian log signer.")
```